src/components/layout/OrgLayout.tsx

```diff
@@ -13,49 +13,69 @@ import OrgHeader from './OrgHeader';
 
 interface OrgLayoutProps {
     children: JSX.Element;
+    mainPage: boolean;
     orgId: string;
 }
 
-const OrgLayout = ({ children, orgId } : OrgLayoutProps) : JSX.Element => {
+const OrgLayout = ({ children, mainPage, orgId } : OrgLayoutProps) : JSX.Element => {
     const orgQuery = useQuery(['org', orgId], getOrg(orgId));
     const router = useRouter();
     const path =  router.pathname.split('/');
     let currentTab = path.pop();
 
-    if (currentTab === '[orgId]') {
-        currentTab = 'home';
-    }
+    if (mainPage) {
 
-    const onSelectTab = (key : ReactText) : void => {
-        if (key === 'home') {
-            router.push(`/o/${orgId}`);
-        }
-        else {
-            router.push(`/o/${orgId}/${key}`);
+        if (currentTab === '[orgId]') {
+            currentTab = 'home';
         }
-    };
 
-    const tab1 = <Msg id="layout.org.tabs.home"/>;
-    const tab2 = <Msg id="layout.org.tabs.comingUp"/>;
-    const tab3 = <Msg id="layout.org.tabs.campaigns"/>;
+        const onSelectTab = (key : ReactText) : void => {
+            if (key === 'home') {
+                router.push(`/o/${orgId}`);
+            }
+            else {
+                router.push(`/o/${orgId}/${key}`);
+            }
+        };
 
+        const tab1 = <Msg id="layout.org.tabs.home"/>;
+        const tab2 = <Msg id="layout.org.tabs.comingUp"/>;
+        const tab3 = <Msg id="layout.org.tabs.campaigns"/>;
+
+        return (
+            //TODO: Break out into separate meta layout component
+            <DefaultLayout org={ orgQuery.data! }>
+                <OrgHeader org={ orgQuery.data! }/>
+                <Tabs
+                    aria-label="Organization submenu"
+                    onSelectionChange={ onSelectTab }
+                    selectedKey={ currentTab }>
+                    <Item key="home" title={ tab1 }>
+                        <Content>{ children }</Content>
+                    </Item>
+                    <Item key="events" title={ tab2 }>
+                        <Content>{ children }</Content>
+                    </Item>
+                    <Item key="campaigns" title={ tab3 }>
+                        <Content>{ children }</Content>
+                    </Item>
+                </Tabs>
+            </DefaultLayout>
+        );
+    }
     return (
+        //TODO: Break out into separate meta layout component
```

Comment:
I'm not entirely sure what this comment refers to, but if it means what I think it means I agree, and I think it should be done as part of this PR. See other comments.